torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2793,6 +2793,11 @@ def call_function(
         args: "list[VariableTracker]",
         kwargs: "dict[str, VariableTracker]",
     ) -> "VariableTracker":
+        from .torch_function import can_dispatch_torch_function, dispatch_torch_function
+
+        if can_dispatch_torch_function(tx, args, kwargs):
+            return dispatch_torch_function(tx, self, args, kwargs)
+
```

Comment:
seems reasonable to me, but (1) can we make sure that in eager-mode, flexattention has a torch_function and (2) can we test that in both eager and compile, flexattention torch_function gets dispatched?